bundles/net.tourbook/src/net/tourbook/ui/views/rawData/RawDataView.java

```diff
@@ -4585,6 +4589,10 @@ private void restoreState() {
       final boolean isIgnoreInvalidFile = _state.getBoolean(STATE_IS_IGNORE_INVALID_FILE);
       _rawDataMgr.setState_IsIgnoreInvalidFile(isIgnoreInvalidFile);
 
+      // restore: set body weight status before the tours are imported
+      final boolean isSetBodyWeight = _state.getBoolean(STATE_IS_SET_BODY_WEIGHT);
```

Comment:
`_state.getBoolean(...` returns false when not yet saved

Reply:
Fixed